pyproject.toml

```diff
@@ -22,8 +22,11 @@ exclude = '''
 '''
 
 [tool.poetry]
-name = "pinecone"
+name = "pinecone-client"
```

Comment:
Interesting. I went back and forth on this name value in my original Poetry PR from `pinecone-client` back to `pinecone` to get everything working.